sage-ios/SAGE/SAGE/AdminOperations.swift

```diff
@@ -99,6 +99,38 @@ class AdminOperations {
         
     }
     
+    static func loadSchool(id: Int, completion: ((School) -> Void), failure: (String) -> Void) {
+        let requestURL = StringConstants.kEndpointBaseURL + "/schools/" + String(id)
+        let manager = BaseOperation.manager()
+        manager.GET(requestURL, parameters: nil, success: { (operation, data) -> Void in
+            // stuff
+            let immutableSchoolDict = (data as! NSDictionary)["school"] as! NSDictionary
+            let schoolDict = immutableSchoolDict.mutableCopy()
+            let userDict = schoolDict["users"] as! NSMutableArray
+            schoolDict.removeObjectForKey("users")
+            var schoolSwiftDict = [String: AnyObject]()
+            for key in schoolDict.allKeys {
+                schoolSwiftDict[key as! String] = schoolDict[key as! String]
+            }
+            let school = School(propertyDictionary: schoolSwiftDict)
+            var students = [User]()
+            for user in userDict {
+                let userDict = user as! NSMutableDictionary
+                var userSwiftDict = [String: AnyObject]()
+                for key in userDict.allKeys {
```

Comment:
u sure thats needed? we never needed that before

On Sun, Nov 29, 2015 at 5:33 PM, Sameera Vemulapalli <
notifications@github.com> wrote:

> In sage-ios/SAGE/SAGE/AdminOperations.swift
> <https://github.com/calblueprint/sage-mobile/pull/56#discussion_r46103027>
> :
>
> > +        manager.GET(requestURL, parameters: nil, success: { (operation, data) -> Void in
> > +            // stuff
> > +            let immutableSchoolDict = (data as! NSDictionary)["school"] as! NSDictionary
> > +            let schoolDict = immutableSchoolDict.mutableCopy()
> > +            let userDict = schoolDict["users"] as! NSMutableArray
> > +            schoolDict.removeObjectForKey("users")
> > +            var schoolSwiftDict = [String: AnyObject]()
> > +            for key in schoolDict.allKeys {
> > +                schoolSwiftDict[key as! String] = schoolDict[key as! String]
> > +            }
> > +            let school = School(propertyDictionary: schoolSwiftDict)
> > +            var students = [User]()
> > +            for user in userDict {
> > +                let userDict = user as! NSMutableDictionary
> > +                var userSwiftDict = [String: AnyObject]()
> > +                for key in userDict.allKeys {
>
> dis to convert from nsmutabledictionary to swift dictionary so we can pass
> into init(propertyDictionary:) methods
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/calblueprint/sage-mobile/pull/56/files#r46103027>.
>